test/src/loaders/workspace_uploader/workspace_uploader_test.py

```diff
@@ -88,7 +88,7 @@ def test_read_upload_status_yaml_file(setup_and_teardown):
         "CI", 12345, "214", assembly_dir, assembly_name
     )
     expected_data = {
-        "CI": {12345: {"file_name": assembly_name, "loads": {}}}
+        "CI": {12345: {assembly_name: {workspace_uploader._KEY_LOADS: {}}}}
```

Comment:
```suggestion
        "CI": {12345: {assembly_name: {"loads": {}}}}
```

We should use literals as much as possible in tests. If someone fatfingers `loads` to `loaeds`, for example, we want the tests to break as that means the code will behave unexpectedly when dealing with existing files

Reply:
Same thing in other places

Reply:
👍

Reply:
The original is like this:
```
CI:
  69046:
    file_name: GCF_001559985.1_GG12_C01_04_genomic.fna.gz
    loads:
         12:
            upa: '69046_311_1'
         13:
            upa: "69046_400_1"
```
It seems you changed file_name as a key now and loads as its val.

Reply:
yea. now for a source genome, we might upload genomic.gbff.gz file as Genome in addition to Assembly. One file_name per genome no longer works.

Reply:
If we add more meta data inside other than file_name in future, original structure seems more flexible to extend.

Reply:
so we need to consider 4 cases?
1. empty
2. assembly, no genome
3. no assembly, genome
4. both

Reply:
it's won't support multiple files, unless I misunderstand the code. If we upload `xxx.genomic.fna.gz` as Assembly first and then `xxx.genomic.gbff.gz` as Genome. `file_name` will be overwritten. And `loads` associated with `file_name` are mixed from Assembly and Genome.

Reply:
will not overwrite because is diff name right? They get diff extensions.

Reply:
wait a second, are you saying both xxx.genomic.fna.gz and xxx.genomic.gbff.gz in the same folder?

Reply:
yea they are downloaded in the same folder.

Reply:
Let's say we add the genome. On the same load id, we want to add the corresponding assembly in that folder. What will happen?

Reply:
I am not sure I get what you mean. I don't think we are adding Assembly to Genome. We now supports both Genome uploads and Assembly uploads. They are completely separate. Each object type should have its own load ids.

Reply:
with this PR, if you uploaded both objects, the yml file should looks like:
```
env:
  ws_id:
      xxx.gbff.gz:
            loads:
                1:
                    upa: 'xxx'
                2:
                   upa: 'xxx'
     xxx.fna.gz:
             loads:
                 1: 
                    upa: 'xxx'
```
But with current structure with `file_name` as a key, we cannot support both files.

Reply:
with this PR, if you uploaded both objects, the yml file should looks like:
```
env:
  ws_id:
      xxx.gbff.gz:
            loads:
                1:
                    upa: 'xxx'
                2:
                   upa: 'xxx'
     xxx.fna.gz:
             loads:
                 1: 
                    upa: 'xxx'
```
But with current structure with `file_name` as a key, we cannot support both files.

Reply:
I somehow feel genome and assembly are not completely separate. If we upload genome already, maybe not need to redo the upload of assembly? Need some clarification here @MrCreosote 

If genome and assembly are completely separate you current design would work.

Reply:
@Xiangs18 @MrCreosote updated file structure per our discussion.